extensions/3DTILES_implicit_tiling/0.0.0/schema/subtree.schema.json

```diff
@@ -19,17 +19,17 @@
       }
     },
     "tileAvailability": {
-      "description": "The availability of tiles in the subtree. If not all available or unavailable, the size of the buffer will be (N ^ (subtreeDepth + 1) - 1) / (N - 1) bits where N is 4 for tiling scheme `QUADTREE` and 8 for `OCTREE`.",
+      "description": "The availability of tiles in the subtree. If not all available or unavailable, the size of the buffer will be (N ^ (subtreeLevels + 1) - 1) / (N - 1) bits where N is 4 for tiling scheme `QUADTREE` and 8 for `OCTREE`.",
```

Comment:
@ErixenCruz I think this formula has an off-by-one error due to how `subtreeLevels` is defined (note that the terminology changed recently) I think it should just be `subtreeLevels` without the +1

Here's my thinking:

| `subtreeLevels` | bits |
|----|---|
| 1 | 1|
| 2 | 1 + 4 = 5 |
| 3 | 1 + 4 + 16 = 21 |
| L | sum for k in `[0, L -1]` of N^k = (N^L - 1) / (N - 1) |

Could you double check this? Then I'll update the schema and tables in the spec.